src/es/components/atoms/arrow/Arrow.js

```diff
@@ -66,12 +71,69 @@ export default class Arrow extends Hover() {
   }
 
   /**
-   * renders the css
+   * Renders the CSS.
    *
    * @return {void}
    */
   renderCSS () {
-    this.css = /* css */`
+    const upKeyframes = /* css */ `
+      @keyframes move {
+        0% { transform: rotate(270deg) translateY(0); }
+        48% { transform: rotate(270deg) translateY(-0.8em); opacity: 1; }
+        49% { transform: rotate(270deg) translateY(-0.8em); opacity: 0; }
+        50% { transform: rotate(270deg) translateY(0.8em); opacity: 0; }
+        51% { transform: rotate(270deg) translateY(0.8em); opacity: 1; }
+        100% { transform: rotate(270deg) translateY(0); }
+      }
+    `
+    const downKeyframes = /* css */ `
+      @keyframes move {
+        0% { transform: rotate(90deg) translateY(0); }
+        48% { transform: rotate(90deg) translateY(-0.8em); opacity: 1; }
+        49% { transform: rotate(90deg) translateY(-0.8em); opacity: 0; }
+        50% { transform: rotate(90deg) translateY(0.8em); opacity: 0; }
+        51% { transform: rotate(90deg) translateY(0.8em); opacity: 1; }
+        100% { transform: rotate(90deg) translateY(0); }
+      }
+    `
+    const leftKeyframes = /* css */ `
+      @keyframes move {
+        0% { transform: rotate(180deg) translateY(0); }
+        48% { transform: rotate(180deg) translateY(-0.8em); opacity: 1; }
+        49% { transform: rotate(180deg) translateY(-0.8em); opacity: 0; }
+        50% { transform: rotate(180deg) translateY(0.8em); opacity: 0; }
+        51% { transform: rotate(180deg) translateY(0.8em); opacity: 1; }
+        100% { transform: rotate(180deg) translateY(0); }
+      }
+    `
+    const rightKeyframes = /* css */ `
+      @keyframes move {
+        0% { transform: translateY(0); }
+        48% { transform: translateY(-0.8em); opacity: 1; }
+        49% { transform: translateY(-0.8em); opacity: 0; }
+        50% { transform: translateY(0.8em); opacity: 0; }
+        51% { transform: translateY(0.8em); opacity: 1; }
+        100% { transform: translateY(0); }
+      }
+    `
+
+    let directionKeyframes = rightKeyframes
```

Comment:
the directionKeyframes string should be applied to this.css = directionKeyframes after the switch, or did I miss something?

Reply:
🙈  Jip, this is right and I forget it. Therefore, I put [more examples in Storybook for Arrow](https://github.com/mits-gossau/web-components-toolbox-storybook/commit/068fa8bc8dfb73fb58dca902c18fd1e4fbaa33a7). Muah!